lib/SPIRV/libSPIRV/spirv.hpp

```diff
@@ -46,7 +46,7 @@ namespace spv {
 
 typedef unsigned int Id;
 
-#define SPV_VERSION 10000
+#define SPV_VERSION 0x10000
 #define SPV_REVISION 2
```

Comment:
I had customer request to supply macros so they could #ifdef their code.  Seemed reasonable enough, as normal C++ if()-else won't work across versions with spelling changes.